CMakeLists.txt

```diff
@@ -77,9 +77,9 @@ elseif(${TRITON_TENSORFLOW_VERSION} EQUAL "2")
   set(TRITON_TENSORFLOW_BACKEND_LIBNAME triton_tensorflow)
   set(TRITON_TENSORFLOW_BACKEND_INSTALLDIR ${CMAKE_INSTALL_PREFIX}/backends/tensorflow)
   if(RHEL_BUILD)
-    set(TRITON_TENSORFLOW_PYTHON_PATH "/opt/_internal/cpython-3.10.13/lib/python3.10/site-packages/tensorflow")
+    set(TRITON_TENSORFLOW_PYTHON_PATH "/opt/_internal/cpython-3.12.1/lib/python3.12/site-packages/tensorflow")
```

Comment:
Is python patch version different for RHEL? I think it was 3.12.3 in ubuntu24

Reply:
I take the path out of container shared with us by upstream.